### ISSUE TITLE
Entry resend fails to send conditional fields

### ISSUE THREAD

**Issue by antipole2** [CalderaWP/Caldera-Forms#2797]
## What Version Of Caldera Forms, WordPress and PHP Are You Using?
Version 1.7.4 

## Does Your Issue Persist When You Disable All Other Plugins and Switch To The Default Theme?
Not tested

I have fields that are displayed conditionally.  The first field is a Terms and Conditions acceptance.  When this box is ticked, the rest of the form discloses.

## What Is The Unexpected Behaviour?
The email sent on submission is correct.  However, a resend request only sends the first field.
(Also the screen returns to the Caldera Forum > Forms page - rather than staying on the form in question.). See the emails included below.  It looks like the resend process trips up over the conditional field.

## What PHP Errors Have You Logged While Reproducing This Bug?
No PHP errors logged

NB I have the Validate Email processor installed.

**Email automatically sent on submission starts:**
T&C

I accept the terms & conditions and the privacy policy
Your known name {o}

Fred
Surname {s}

Smith
Your email address {s}

<Deleted here>
Fall-back number to text {s}

+1
Account type

Yourself & yacht
Yacht name {o}

Titanic

**On resend the email reads only:**
T&C

I accept the terms & conditions and the privacy policy, {"opt1591954":"I accept the terms & conditions and the privacy policy"}

**Comment by swapnilmanjrekar** [comment]
I tried the following steps to check this Bug, I installed WordPress locally using Lando on my desktop, I Created a test site, on it. I created a blank form using CF plugin, added a verify email addon and verify email processor, I added a consent field with a checkbox then i added "Your known Name" name field, "Surname" name field, "Phone number" field, "Email address" field, created an "Account type" field using dropdown select field with 3 options, created "Yacht Name" field using dropdown select field with 3 options, added a submit button to submit the form then  added a condition that once the T&C field is checked other fields are disclosed on the form . When I Submitted the form, I received a verification email on the entered email address and once i verified the email the form was submitted successfully & I received an email info of the submitted form as follows:
Email automatically sent on submission starts:
T&C

I accept the terms & conditions and the privacy policy
Your known name {o}

Chris
Surname {s}

Martin
Your email address {s}

chris@gmail.com

Call-back number to text {s}

+1234564426

Account type

Yourself & yacht

Yacht name {o}

Titanic
 
Also when I go to CF menu in my WordPress Test site Dashboard, Clicked on my created T&C form and in the Entries options when I clicked on the resend option I receive the resend email as follows:
T&C

I accept the terms & conditions and the privacy policy
Your known name {o}

Chris
Surname {s}

Martin
Your email address {s}

chris@gmail.com

Call-back number to text {s}

+1234564426

Account type

Yourself & yacht

Yacht name {o}

Titanic

So this email that I received is exactly the same email when the customer submitted the form and verified his email address. Also on the console of my browser, I did not see any javascript errors and also in the debug.log file of my test site root directory I didn't find any PHP errors. So According to my testing and findings, I do not see this as a "Bug"

**Comment by antipole2** [comment]
Mmm.. thanks for looking into this.  I am still seeing the problem.  I have now done the following:

Using the Health Check plugin, I entered troubleshooting mode and enabled only the plugin Caldera Forms.  This also switched to the Twenty Sixteen theme.

I viewed an existing entry logged in Caldera Forms (it looks fine) and then resent it. The email shows just the T&C bit as reported above.  I submitted a new form entry using a simpler branch of the form and saw the same issue.

Possibly this is something about my form that your test did not reproduce?  I could export and send my form to a tester.

regards, Antipole

WordPress Version  4.9.8; PHP Version 5.6.38; Caldera Forms Version 1.7.4
